libs/core/src/lib/checkbox/checkbox/checkbox.component.ts

```diff
@@ -40,6 +40,37 @@ export class CheckboxComponent implements ControlValueAccessor {
     @ViewChild('labelElement')
     labelElement: ElementRef;
 
+    /**
+     * Attached to the aria-label attribute of the host element. In most cases, aria-labelledby will
+     * take precedence so this may be omitted.
+     */
+    @Input('aria-label')
+    ariaLabel = '';
+
+    /**
+     * Users can specify the `aria-labelledby` attribute which will be forwarded to the input element
+     */
+    @Input('aria-labelledby')
+    ariaLabelledby = null;
+
+    /** The 'aria-describedby' attribute is read after the element's label and field type. */
+    @Input('aria-describedby')
+    ariaDescribedby: string;
+
+    /** The 'aria-disabled' for giving accessibility for disabled checkbox element. */
+    @Input('aria-disabled')
+    ariaDisabled: boolean;
+
+    /** sets checkbox tooltip */
+    @Input()
+    title: string;
+
+    /**
+     * Includes the checkbox in the page tab sequence.
+     */
+    @Input()
+    tabIndex = 0;
```

Comment:
Let's switch  this to @Attribute decorator, as tab index will have always constant value.

Reply:
using @Attribute in constructor now.
Done